tests/queries/0_stateless/00700_decimal_with_default_precision_and_scale.sql

```diff
@@ -0,0 +1,19 @@
+DROP TABLE IF EXISTS decimal;
+
+CREATE TABLE IF NOT EXISTS decimal
+(
+    d1 DECIMAL(9, 8),
+    d2 DECIMAL(18),
+    d3 DECIMAL
+)
+ENGINE = MergeTree
+PARTITION BY toInt32(d1)
+ORDER BY (d2, d3);
+
+INSERT INTO decimal (d1, d2, d3) VALUES (4.2, 4.2, 4.2);
+
+SELECT type FROM system.columns WHERE table = 'decimal' AND database = currentDatabase();
```

Comment:
Should have put in an `ORDER BY` to avoid flaky tests but I can do that in a separate PR.